Part 1 - Personal Blog/post.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/7dab968d48.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Seb Lex's Personal Blog</title>
+</head>
+
+<body>
+    <nav class="top-nav">
+        <div class="container">
+            <div class="logo">Seb Lex's Blog</div>
+            <div>
+                <ul class="nav">
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+
+    <div class="content container">
+        <div class="cards">
+            <article class="card">
+                <h2>Article 1</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laboriosam ullam natus culpa harum, fugit
+                    reiciendis similique! Quaerat nostrum, ipsum aspernatur id vel deleniti officia consequatur
+                    consectetur qui sequi eius quia.</p>
+
```

Comment:
Rubric ask to add typography selector in article.

Reply:
Done.